### PR TITLE
Add star history kadi app

### PR DESCRIPTION
**Pull request by jeanconn**
Add star history kadi app

This is intended to replace

https://icxc.harvard.edu/cgi-bin/aspect/get_stats/get_stats.cgi?id=118494392

### CODE REVIEW

File: mica/web/star_hist.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from astropy.table import Table
+from mica.stats import acq_stats
+from Ska.DBI import DBI
+
+ACQ_COLS = ['date', 'type', 'obsid', 'obi', 'slot', 'mag',
+            'mag_obs', 'id', 'stat_flags', 'star_tracked', 'acq_dy', 'acq_dz']
+
+GUI_COLS = ['date', 'type', 'obsid', 'obi', 'slot', 'mag',
+            'mag_obs', 'perc_not_track']
```
Comment (jeanconn) [inline mica/web/star_hist.py]: This column defs should be cut as they aren't used.

**Comment by taldcroft** [comment]
Some description on what this is providing as well as function docstrings and code comments would be good.

**Comment by jeanconn** [comment]
I probably don't need the star_hist code at all, but my templating isn't great, so just making lists of dicts of what I want is a cheap cheat.

**Comment by jeanconn** [comment]
It would be better for me to store the column order and formatting in this code instead of in the template, but this is probably fine for a version 1.

**Comment by jeanconn** [comment]
I should add time range filtering; that will let me use the app to cover and display whatever range we want in starcheck.

**Comment by jeanconn** [comment]
Should also add some notes to the page describing what some of the more ambiguous acquisition columns (star_tracked, dy, dz) are.

**Comment by taldcroft** [comment]
Looks good to me!

**Comment by taldcroft** [comment]
This particular code doesn't yet need to run or be tested against Py3, but it's worth checking that it is at least syntax compatible with a spin through 2to3.

**Pull request merged**